.circleci/config.yml

```diff
@@ -30,3 +31,20 @@ jobs:
           ./scripts/wait-for-local-keycloak.sh
           ./scripts/create-terraform-client.sh
           make testacc
+  client-credentials:
+    <<: *test-template
+    environment:
+      GO111MODULE: "on"
+      KEYCLOAK_CLIENT_ID: terraform
+      KEYCLOAK_URL: http://localhost:8080
+      KEYCLOAK_REALM: master
+      KEYCLOAK_CLIENT_SECRET: 884e0f95-0f42-4a63-9b1f-94274655669e
+  password:
+    <<: *test-template
+    environment:
+      GO111MODULE: "on"
+      KEYCLOAK_CLIENT_ID: admin-cli
+      KEYCLOAK_URL: http://localhost:8080
+      KEYCLOAK_REALM: master
+      KEYCLOAK_USER: keycloak
+      KEYCLOAK_PASSWORD: password
```

Comment:
I like this idea.  I'll have to update the branch protection to require these new statuses set by CircleCI.